aws/table_aws_account_contact.go

```diff
@@ -0,0 +1,163 @@
+package aws
+
+import (
+	"context"
+
+	go_kit_packs "github.com/turbot/go-kit/types"
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+
+	"github.com/aws/aws-sdk-go-v2/service/account"
+	"github.com/aws/aws-sdk-go-v2/service/account/types"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsAccountContact(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_account_contact",
+		Description: "AWS Account Contact",
+		List: &plugin.ListConfig{
+			Hydrate: getAwsAccountContact,
```

Comment:
```suggestion
			Hydrate: listAwsAccountContacts,
```
We can only ever get one, but to be consistent, we should name this list function like the others